### ISSUE TITLE
Webkit-Scrollbar Pseudo class is invalid

### ISSUE THREAD

**Issue by BenedictLang** [parcel-bundler/parcel#7919]
# 🐛 bug report

Parcel does not want me to use a pseudo class on webkit scrollbar thumb.

## 🎛 Configuration (.babelrc, package.json, cli command)

```js
"source": "src/index.html",
  "scripts": {
    "start": "parcel",
    "prod": "parcel build --no-source-maps"
  },
  "devDependencies": {
    "@parcel/optimizer-data-url": "^2.4.1",
    "@parcel/packager-raw-url": "^2.4.1",
    "@parcel/transformer-inline-string": "^2.4.1",
    "@parcel/transformer-sass": "^2.4.1",
    "@parcel/transformer-webmanifest": "^2.4.1",
    "parcel": "^2.4.1",
    "sass": "^1.38.1"
  },
  "dependencies": {
    "npm": "^8.6.0"
  }
```

## 🤔 Expected Behavior

Stylable scrollbar thumb on hover.

## 😯 Current Behavior

```
× Build failed.

@parcel/transformer-css: Invalid pseudo class after ::-webkit-scrollbar pseudo element

  D:\_Projekte\Websites\-------\src\styles\main.scss:1058:27
    1057 |
  > 1058 | ::-webkit-scrollbar-thumb:hover {
  >      |                           ^
    1059 |   background-color: var(--clr-text);
    1060 | }

```

## 💁 Possible Solution

Saw in the changelog of v2.4.1 some changes on this pseudo classes, might be an issue there...

## 🔦 Context

Trying to style the scrollbar like I did in other parcel v2.0 projects too.

## 💻 Code Sample

```
/*================================== SCROLLBAR ==================================*/
$clr-text: rgb(255, 255, 255);
$clr-third: rgba(17, 23, 37, .25);
$clr-background: rgba(41, 45, 66, 1);

::-webkit-scrollbar {
    width: clamp(5px, 0.6rem + 6vw, 1rem);
}

::-webkit-scrollbar-track {
    background: $clr-background;
    border-left: 10px solid transparent;
}

::-webkit-scrollbar-thumb {
    border-radius: 10px;
    border: 4px solid transparent;
    opacity: 0.2;
    background-color: $clr-third;
    background-clip: content-box;
}

::-webkit-scrollbar-thumb:hover{
    background-color: $clr-text;
}
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | v2.4.1 |
| Node             | v14.17.6 |
| npm/Yarn         | v8.6.0 |
| Operating System | Win11 |

_**PS:** Strange is, that parcel now copies all files together to the main and telling me than which lines of code have issues, so I have to search in every imported file, where the code actually accures (see error log above, I do not have 1000 lines scss in one file...in this case easy bc i had scrollbar styling in a separate file)._

**Comment by devongovett** [comment]
Duplicate of #7885. Fixed by https://github.com/parcel-bundler/parcel-css/commit/68aa48d7d041012e04e0c10071cd20822de0d655. Will be released shortly.

**Comment by devongovett** [comment]
Fixed in `@parcel/css` v1.8.0. You may need to remove the previous version from your lock file to update.

**Comment by BenedictLang** [comment]
> Fixed in `@parcel/css` v1.8.0. You may need to remove the previous version from your lock file to update.

oh yea, its on 1.7.4
Thank you!
